src/App.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Sum from './components/Sum';
+import './App.scss';
+
+const App = () => (
+  <>
+    <h1>React sum</h1>
+    <Sum
+      a={2}
+      b={2}
+    />
```

Comment:
It is not necessary to add new lines here. It is short enough.

```suggestion
    <Sum a={2} b={2} />
```